homeassistant/components/nextcloud/entity.py

```diff
@@ -21,3 +23,14 @@ def __init__(self, coordinator: NextcloudDataUpdateCoordinator, item: str) -> No
     def unique_id(self) -> str:
         """Return the unique ID for this sensor."""
         return f"{self.coordinator.url}#{self.item}"
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return device specific attributes."""
+        return DeviceInfo(
+            name=self.coordinator.url,
+            identifiers={(DOMAIN, self.coordinator.url)},
```

Comment:
What if the URL changes?

Reply:
I'm not sure if a NextCloud server provides a unique-id.
If it doesn't, then maybe using the `config_entry.entry_id` is best as identifier.

Reply:
as far as i know, there is no uid provided by nextcloud, so I would go with `config_entry.entry_id`